src/components/Search/types.ts

```diff
@@ -45,6 +45,7 @@ type SelectedReports = {
     reportID: string;
     policyID: string | undefined;
     action: ValueOf<typeof CONST.SEARCH.ACTION_TYPES>;
+    allActions: Array<ValueOf<typeof CONST.SEARCH.ACTION_TYPES>>;
```

Comment:
BTW I don't think this was the correct solution. We should not have added this key to Onyx

Reply:
`SelectedReports` is not an Onyx data. It's just a derived value.